products.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE HTML>
 <html>
 	<head>
-	<title>Trabajo Final</title>
+	<title>Trabajo Final con cambios</title>
```

Comment:
me gustó mucho el título
```suggestion
	<title>Trabajo Final con cambios</title>
```